subsys/bluetooth/audio/Kconfig.hap

```diff
@@ -0,0 +1,11 @@
+# Bluetooth Audio - Hearing Access Profile options
+#
+# Copyright (c) 2023 Codecoup
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_HAP_HARC
+	bool "HAP Hearing Access Remote Controller support [EXPERIMENTAL]"
+	depends on BT_CSIP_SET_COORDINATOR
```

Comment:
The HAP HARC has additional requirements as well: 
![image](https://github.com/zephyrproject-rtos/zephyr/assets/2520413/d0086e05-a9a9-43e2-a897-2d2f1114da0a)

Reply:
Yes, but on the other hand, it does not relate to neither VCP Volume Controller nor MICP Microphone Controller. There is no procedure that would describe the dependencies...

Reply:
@MariuszSkamra those dependencies come from the CAP Commander dependency :) It would appear that the HARC indirectly has a dependency to support the Change Volume procedure from CAP (added by https://github.com/zephyrproject-rtos/zephyr/pull/65214). 

Of course, that implicit requirement/dependency should arguably have been made explicit in the HAP spec.